tool/grind.dart

```diff
@@ -80,12 +80,12 @@ build() {
 
   // Run vulcanize.
   // Imports vulcanized, not inlined for IE support
-  vulcanizeNoExclusion('scripts/imports.html');
+/*  vulcanizeNoExclusion('scripts/imports.html');
```

Comment:
This makes travis go green, but will cause problems when we try and build the front-end locally.